packages/lib/src/form-fields/file-list/file-list.wc.svelte

```diff
@@ -46,6 +47,16 @@
   let hoveringFile = false;
   let fileElements = [];
   let internalValue = '';
+  let showCropper = false;
+  let beingCropped: number = null;
+  let mandatoryCroped = 0;
+  enum cropOptions {
+    disabled = 0,
+    optional = 1,
+    mandatory = 2
+  }
+  export let cropperEnable: cropOptions.disabled | cropOptions.mandatory | cropOptions.optional =
+    cropOptions.disabled;
```

Comment:
It would be better if the cropOptions attribute is defined by a set of options that are type string.